.github/workflows/labeler-deploy-main.yml

```diff
@@ -12,10 +12,10 @@ on:
       - pyproject.toml
 
 jobs:
-  label-deployment-dev:
+  label-deployment-main:
     runs-on: ubuntu-latest
     steps:
       - name: add-label
         uses: andymckay/labeler@master
         with:
-          add-labels: "deployment-dev"
+          add-labels: "deployment-main"
```

Comment:
Let's actually keep this label as `deployment-dev` and the workflow name / job name the same. It is a marking a deployment to the `dev` environment.

But we do need to change the branch as you've done here.